content/developer/reference/backend/actions.rst

```diff
@@ -445,3 +437,47 @@ Actions triggered automatically on a predefined frequency.
 
 ``nextcall``
     Next planned execution date of this action (date/time format)
+
+``priority``
+    Priority of the action when executng multiple action at the same time
+
+
+Advanced use: Batching
+----------------------
+
+When executing an scheduled action, it's recommended to try batching progress in order
+to avoid hogging a worker for a long period of time and possibly running into timeout exceptions.
+
+Odoo provide a simple API for scheduled action batching;
+
+.. code-block:: python
+
+      self.env['ir.cron']._notify_progress(done=XX:int, remaining=XX:int)
+
+This method allow the scheduler to know if progress was done and whether there is
+still remaining work that need to be done.
+
+By default, if the api is used, the scheduler try to process 10 batches in one sitting.
+If there is still remaining tasks after those 10 batches, a new cron call will be executed as
+soon as possible.
+
+Advanced use: Triggers
+----------------------
+
+For more complex use case, Odoo provide a more advanced way to trigger
+scheduled actions from python directly.
+
+.. code-block:: python
+
+      action_record._trigger(at=XX:date)
+
+Security
+--------
+
+To avoid a fair usage of resources among scheduled actions, some security measure ensure the
+correct functioning of your scheduled actions.
+
+- If an scheduled action err or timeout 3 consecutive times,
+  it will skip it's current execution and be considered as failed.
+- If an scheduled action fail it execution 5 times consecutively over a period of at least
+  7 days, it will be deactivated and will notify the DB admin.
```

Comment:
```suggestion
To avoid a fair usage of resources among scheduled actions, some security measures ensure the
correct functioning of your scheduled actions.

- If a scheduled action encounters an error or a timeout three consecutive times,
  it will skip its current execution and be considered as failed.
- If a scheduled action fails its execution five consecutive times over a period of at least
  seven days, it will be deactivated and will notify the DB admin.
```